Sources/StreamChat/Controllers/ChannelListController/ChannelListController.swift

```diff
@@ -93,6 +93,29 @@ public class ChatChannelListController: DataController, DelegateCallable, DataSt
         return observer
     }()
     
+    lazy var updatedChannelObserver: ListDatabaseObserver<ChatChannel, ChannelDTO> = {
+        let request = NSFetchRequest<ChannelDTO>(entityName: ChannelDTO.entityName)
+        request.sortDescriptors = [ChannelListSortingKey.defaultSortDescriptor]
+        // Channels which are not linked to this query
+        request.predicate = NSCompoundPredicate(
+            notPredicateWithSubpredicate: NSPredicate(
+                format: "ANY queries.filterHash == %@", query.filter.filterHash
+            )
+        )
```

Comment:
Would be nice to have this predicate declared in `ChannelDTO` next to other ones 👍